clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/images/memory/MemoryCreator.kt

```diff
@@ -0,0 +1,64 @@
+package com.clevertap.android.sdk.inapp.images.memory
+
+import android.graphics.Bitmap
+import com.clevertap.android.sdk.ILogger
+import java.io.File
+
+class MemoryCreator {
```

Comment:
this can be an object.

Reply:
Idea was to create static kind of methods equivalent to factory pattern and not singleton. Kotlin docs https://kotlinlang.org/docs/object-declarations.html#companion-objects and other blogs shows companion object way of creating factories. It's still debatable keeping as it is as of now.